libs/domain/search/facet-value-navigation/src/facet-value-navigation.component.ts

```diff
@@ -45,7 +45,7 @@ export class SearchFacetValueNavigationComponent
   }
 
   protected override render(): TemplateResult {
-    return html` <oryx-collapsible ?open=${this.open}>
+    return html` <oryx-collapsible ?open=${this.open} nonTabbable>
```

Comment:
I like this solution! What about `nonTabbable=${onlyWhenTheClearButtonIsAvailable}` ?